### DISCUSSION TITLE
calibre: cannot detect connected devices (private-dev)

### DISCUSSION THREAD

**Discussion by master-bob** [netblue30/firejail#5835] [Q&A] [answered]
Hello Esteemed Community,

To begin with the details of the current configuration.

- Debain 11
- Calibre 6.17
- firejail 0.9.72

Whenever I run Calibre through firejail, calibre is unable to detect connected devices. 

If I go directly to the installed location `/opt/calibre` and run `./calibre`, calibre will automatically detect my connected device.

I'm not sure how to resolve this, besides just running calibre outside of firejail.

I have searched for issues related to this, but did not find any as I don't have the device mounted.

Any suggestions?

Thank you for your time.

**Comment by ghost** [comment, marked as answer]
Thanks for supplying details on your configuration. Have you tried disabling the `private-dev` option that's present in /etc/firejail/calibre.profile yet? Create a `~/.config/firejail/calibre.local` file with the below content and try running calibre through firejail again:

```
ignore private-dev
```

You can use the same calibre.local file to remove (ignore foo) other options and try to find what is blocking access to your connected device. In case this doesn't help to fix things we might have to turn this into an issue report, but let's wait for that until you can test the above and report back.

**Reply by master-bob** [reply]
Turning off `private-dev` allows calibre to automagically detect my device.

Thank you for your help!
